web/packages/teleterm/src/ui/services/usage/setUpUsageReporting.ts

```diff
@@ -16,14 +16,23 @@
 
 import { ConfigService } from 'teleterm/services/config';
 import { ModalsService } from 'teleterm/ui/services/modals';
+import { ENV } from 'teleterm/env';
+import Logger from 'teleterm/logger';
 
 export async function setUpUsageReporting(
   configService: ConfigService,
   modalsService: ModalsService
 ): Promise<void> {
+  const logger = new Logger('setUpUsageReporting');
+  if (!ENV.prehogAddress) {
```

Comment:
Just to confirm and have this written down somewhere, the prod address is going to be added before the public release of v12 so it doesn't really matter that 12.0.0 is not going to have it, right?

Reply:
It is hard to say when exactly we add the prod address. It may be before or after the v12 release (reporting team is working on updating the prod env).
And because of this, we disable this whole functionality until reporting backend is ready. 

I'm not sure what do you mean by public release of v12 vs 12.0.0 😬

Reply:
> I'm not sure what do you mean by public release of v12 vs 12.0.0 😬

When a new major version is released, we usually follow it up with marketing material, blog posts and so only a month after release, usually when x.1 is ready. So even though 12.0.0 will be available to download, there won't be a big fanfare about it until a month after.

Reply:
Ah right. I suppose we will be able to enable it in a public release.